src/ast/getAstSchema.ts

```diff
@@ -341,24 +341,20 @@ export const getAstSchema = (
             }),
         StatementTry: (
             statements: A.AstStatement[],
+            catchName: A.AstId | undefined,
+            catchStatements: A.AstStatement[] | undefined,
```

Comment:
Looks like this didn't get updated

Reply:
actually i thought it might be ok here but lets use single type everywhere

Reply:
fixed